currencyConverter.js

```diff
@@ -0,0 +1,66 @@
+var request = require('request');
+var cheerio = require('cheerio');
+
+var roundOff = function(currency){
```

Comment:
Typical JS style is to have a space before an opening bracket. So here you should add 2: 

`function (currency) {`